integration_tests/test_commit_format.py

```diff
@@ -1,16 +1,25 @@
 # Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
-"""Test the commit message format."""
+"""
+Test the commit message format.
+
+This test works properly on the local machine only when the base branch
```

Comment:
Working with origin is a bit of an anti-pattern. For example, locally I have the upstream remote named `upstream`, and `origin` points to my fork such that it is not possible to push to upstream by mistake (this in theory should not be possible because it should be configured in GitHub that you cannot push to master without reviews).

I would like us to be able to override these values with something that is not BUILDKITE_, but we can do that as a subsequent change.

Reply:
Done.